src/legacy/core_plugins/data/public/filter/filter_bar/_index.scss

```
@@ -1,3 +1,4 @@
 @import 'variables';
 @import 'global_filter_group';
 @import 'global_filter_item';
+@import './filter_editor/index';
```

Can you make this path to the file consistent to the ones above by removing the `./`.
```suggestion
@import 'filter_editor/index';
```

---

Done